src/common/sonarqube-v4/__tests__/analyze-task-test.ts

```diff
@@ -48,7 +48,7 @@ it("should run scanner", async () => {
   await analyze.analyzeTask(EndpointType.SonarQube);
 
   expect(Scanner.getAnalyzeScanner).toHaveBeenCalledWith(
-    path.join(__dirname, "..", ".."),
+    path.join(__dirname, ".."),
```

Comment:
shouldn't this be `__dirname`? seeing below
https://github.com/SonarSource/sonar-scanner-vsts/pull/369/files#diff-b61d2ef3250bb55d7f9192dfed56523dcb5e2027ee2e6fcd8f7ef004853eee06R27

we pass the rootpath, am I missing something?

Reply:
We shouldn't because we need to go from `__tests__` folder to parent folder where `__dirname` is executed in `analyze-task.ts`